src/components/SpaceProposalPage.vue

```diff
@@ -244,8 +244,9 @@ onMounted(() => {
           :loaded-results="loadedResults"
           :strategies="strategies"
         />
+        <!-- space?.nfrClaimer.enabled  -->
         <SpaceProposalNFTSidebar
-          v-if="space?.nftMinterEnabled || true"
+          v-if="web3Account"
```

Comment:
The nft claimer should always be visible no? If they are not connected and click claim, it should open the connect wallet modal.

Reply:
For now we need web3Account cos we get supply info directly from smart contract

Reply:
We should still show the claimer, but with some information about connecting the wallet